packages/charts/theme/vaadin-chart-base-theme.js

```diff
@@ -0,0 +1,1027 @@
+/**
+ * @license
+ * Copyright (c) 2015 - 2021 Vaadin Ltd
+ * This program is available under Commercial Vaadin Developer License 4.0, available at https://vaadin.com/license/cvdl-4.0.
+ */
+
+/**
+ * @license Highcharts
+ *
+ * (c) 2009-2016 Torstein Honsi
+ *
+ * License: www.highcharts.com/license
+ */
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+/* When updating this file do not override vaadin-charts custom properties section */
+
+const chartBaseTheme = css`
+  .highcharts-container {
+    position: relative;
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+    text-align: left;
+    line-height: normal;
+    z-index: 0;
+    /* #1072 */
+    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+    font-family: -apple-system, BlinkMacSystemFont, 'Roboto', 'Segoe UI', Helvetica, Arial, sans-serif,
+      'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol';
+    font-size: 12px;
+  }
+
+  :where(div#chart[styled-mode]) .highcharts-root {
```

Comment:
I would assume this would be enough?

```suggestion
  :where([styled-mode]) .highcharts-root {
```

Reply:
It is enough. I applied the change as suggested to all selectors.